src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -581,7 +581,9 @@ def register_observations(self, builder: Builder) -> None:
             INTERVENTIONS.ANTIBIOTICS,
             INTERVENTIONS.PROBIOTICS,
         ]:
-            pop_filter += f" & pregnancy_outcome == '{PREGNANCY_OUTCOMES.LIVE_BIRTH_OUTCOME}'"
+            pop_filter += (
+                f" & {COLUMNS.PREGNANCY_OUTCOME} == '{PREGNANCY_OUTCOMES.LIVE_BIRTH_OUTCOME}'"
+            )
```

Comment:
The way this is written, it looks like "tracked == True" gets removed from the pop filter based on line 578 above. I don't think that really matters but something to be aware of if you run into issues.